paths/accounts/listAccountTransfers.json

```diff
@@ -0,0 +1,76 @@
+{
+  "get": {
+    "tags": [
+      "Accounts",
+      "Transfers"
+    ],
+    "summary": "List Transfers for Account",
+    "description": "List all Transfers that have been created by your Main Account.",
+    "operationId": "listTransfersForAccount",
+    "parameters": [
+      {
+        "in": "path",
+        "name": "id",
+        "description": "UUID of Main Account",
+        "schema": {
+          "type": "string",
+          "format": "uuid"
+        },
+        "required": true
+      }
+    ],
+    "responses": {
+      "200": {
+        "description": "OK",
+        "content": {
+          "application/vnd.dwolla.v1.hal+json": {
+            "schema": {
+              "$ref": "../../schemas/transfers/Transfers.json#/Transfers"
+            },
+            "example": {
+              "_links": {
+                "self": {
+                  "href": "https://api-devint.dwolla.com/accounts/22926523-b517-43b9-959b-6ac4a422d71e/transfers",
```

Comment:
smol detail. Can we update these example links to point to sandbox? `api-sandbox`

Reply:
Oh, whoops, I usually try to change them. The rest of the document uses `https://api.dwolla.com`. Is that okay for this one as well?